spec/units.yaml

```diff
@@ -398,3 +398,11 @@ Ohm:
   unit: Ohm
   quantity: resistance
   allowed-datatypes: ['numeric']
+
+# Illuminance
+
+Lux:
```

Comment:
Historically we have typically used the unit abbreviation/symbol as entry point here, also described at https://covesa.github.io/vehicle_signal_specification/rule_set/data_entry/data_units/

If we would follow that approach I think `lx` would be more correct, see https://en.wikipedia.org/wiki/Lux

Reply:
@mikehaller - to be consistent with most other units I think `lx:` would be better than `Lux:` as we typically use the abbreviation/symbol of the unit. Any opinion?